src/protojure/internal/grpc/client/providers/http2/jetty.clj

```diff
@@ -42,7 +42,10 @@
                  (resolve result))
                (failed [_ error]
                  (reject error)))]
-       (f p)))))
```

Comment:
Im pretty sure this would already catch exceptions since its within a p/promise block